Assets/Scripts/Jump.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Jump : MonoBehaviour {
+	
+	public float jumpSpeed = 8.0f;
+
+	private Vector3 moveDirection = Vector3.zero;
+	private bool grounded = true;
+	private InputController inputController;
+	private int playerNum;
+	private string jumpAxis;
+	// Use this for initialization
+	void Start () {
+		inputController = GameObject.FindWithTag ("GameController").GetComponent<InputController>();
+		string parentTag = this.transform.parent.tag;
+		string lastChar = parentTag.Substring (parentTag.Length - 1);
+		int.TryParse (lastChar, out playerNum);
+		jumpAxis = inputController.GetThrowInput (playerNum);
+	}
+	
+	void FixedUpdate() 
+	{
+		Jumping();
```

Comment:
If `Jump` isn't reserved, should we call `Jumping` `Jump` instead? "Jumping" makes me think it's a misnamed boolean.

Reply:
Jump is reserved